### PR TITLE
Editable events and appearance fixes

### PR DESCRIPTION
**Pull request by bmansell**

### CODE REVIEW

File: partEZ/app/Http/Controllers/EventController.php [hunk 1]
```diff
@@ -66,6 +66,92 @@ public function details($eid)
...
+        $invites = Self::getInvitesFromEid($eid);
+        $all_poll_options = Self::getPollOptionsFromEid($eid);
+        $itemslist = Event::getEventItems($eid);
+        $items = [];
+
+        foreach ($itemslist as $item)
+        {
+            array_push($items, $item);
+        }
+        $chat_messages = MessageController::getMessagesFromEid($eid);
```
Comment (maxewaschuk) [inline partEZ/app/Http/Controllers/EventController.php]: I don't think we need to grab the chat log when editing an event.

Comment (minnerthecat) [inline partEZ/app/Http/Controllers/EventController.php]: I don't think we need to grab the chat log for this page, it doesn't get used at all in this function.

**Comment by JenCabral** [comment]
I agree with what Max said, after that change, it's good to go :+1:

**Comment by JenCabral** [comment]
k, everything looks good now, merging

**Pull request merged**